### PR TITLE
Update fujitsu-scansnap-home from 1.6.0 to 1.7.0

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by core-code** [comment]
ok this is now an installer, which is bad. to continue to use the package we would have to download 
`https://origin.pfultd.com/downloads/IMAGE/driver/ss/inst2/ix1500/m-software/MacSSHomeInstaller_1_7_0.dmg
`
however this requires authentication. their username and password are super-easy to sniff, but i doubt we should put their 'secrets' in the caskfile and i am not even sure if HTTP Basic Authentication works with credentials stored in the URL stanza of the caskfile.

so we need to get the installer working instead.

i'd recommend removing 'container nested' as well as 'pkg' and adding "installer manual: 'ScanSnap Home Setup.app'"

File: Casks/fujitsu-scansnap-home.rb [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-  url "https://origin.pfultd.com/downloads/ss/sshinst/m-#{version.no_dots}/MacSSHOfflineInstaller_#{version.dots_to_underscores}.dmg"
+  url "https://origin.pfultd.com/downloads/ss/sshinst/m-#{version.no_dots}/MacSSHDownloadInstaller_#{version.dots_to_underscores}.dmg"
   name 'ScanSnap Home'
   homepage 'https://www.fujitsu.com/global/products/computing/peripheral/scanners/scansnap/software/sshome/index.html'
 
   depends_on macos: '>= :sierra'
-  container nested: "Download/MacSSHomeInstaller_#{version.dots_to_underscores}.dmg"
 
-  pkg 'ScanSnap Home.pkg'
+  installer manual: 'SSHDownloadInstaller.app'
```
Comment (vitorgalvao) [inline Casks/fujitsu-scansnap-home.rb]: The app’s true file system name.

Comment (core-code) [reply Casks/fujitsu-scansnap-home.rb]: ah yes, sorry i missed that

**Comment by parkerbxyz** [comment]
@suschizu why the change in installation method? This now requires manual installation each time the app is upgraded...

**Comment by core-code** [comment]
@parkerbxyz 

she did not change the installation method - upstream did that. obviously we cannot keep a "pkg" stanza in the caskfile if it actually is an installer now.

reverting to version 1.6.0 is not an option. Homebrew Cask only carries the latest version of software, sticking to some old version is not a solution.

i hate installers as much as you do and i'd suggest sending some feedback to fujitsu to stop that nonsense. and any other vendor too :)

**Comment by suschizu** [comment]
hey @parkerbxyz - sorry for that but I'm not responsible for changing the installation method  - perhaps ask directly the guys from fujitsu (http://scansnap.fujitsu.com/global/dl/) - I just adjusted the cask to the latest release of scansnap - perhaps you can find a download without installer for version 1.7.0 (?)

**Comment by core-code** [comment]
i already mentioned where the download without installer is now, see the comment above.

**Comment by core-code** [comment]
if the "offline installers" (=pkg) are really posted with a time-delay i guess it makes sense to wait for them.

**Comment by parkerbxyz** [comment]
> perhaps you can find a download without installer for version 1.7.0 (?)

@suschizu No worries! The offline installer for the latest release is usually made available approximately two weeks after the initial release date here: http://scansnap.fujitsu.com/global/dl/mac-sshoffline-q.html

> if the "offline installers" (=pkg) are really posted with a time-delay i guess it makes sense to wait for them.

@core-code The offline installer for 1.7.0 is now available and I've updated my PR accordingly: #1520

**Comment by suschizu** [comment]
@parkerbxyz - good to know - thanks!

**Pull request merged**